src/models/Station.tsx

```diff
@@ -0,0 +1,16 @@
+import { Journey } from './Journey';
+
+export type Station = {
+    id: number;
+    name: string;
+    crs: string;
+    nlc: string;
+    latitude: number;
+    longitude: number;
+    value: string;
+    label: string;
+}
+
+// export interface DropdownStationOption {
+//
+// }
```

Comment:
Make sure you have a look at your changes before committing them

Reply:
Changed it now